pkg/imagedetails/imagedetails.go

```diff
@@ -82,6 +82,12 @@ func GetImageDetails(ctx context.Context, c kclient.Client, namespace, imageName
 	if err != nil {
 		return nil, err
 	}
+
+	opts, err = images.GetAuthenticationRemoteOptions(ctx, c, namespace, opts...)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I don't think this piece is necessary. Without this PR, I am already able to run `acorn image details <private image>` and it uses my credentials properly to fetch the information.

Reply:
That's exactly the part I'm unsure about, since that's the point where I think https://github.com/acorn-io/runtime/issues/2026 could come from.
Since in the Manager setup, there won't be local auth for ECR, so without this, we'd have no auth at all to use here.
The other function up there (https://github.com/acorn-io/runtime/pull/2027/files/668bfe95105a7c374ea4049c7deb6c4436a337db#diff-c0f7bfbedf36fddcbdbede8bb9f17a86a8e8bd885e6f4fb2886ccd4c2b190a6aR76) does this within the function body and apparently didn't yield the error from the issue.
I couldn't reproduce the `401` myself, so this is more of a guess and would at least make the remote.Options consistent between `images.PullAppImageWithDataFiles` and `acornsign.FindSignature` :thinking:

Reply:
Okay, that works for me then. Hopefully this will fix the 401.